decoimpact/business/utils/dataset_utils.py

```diff
@@ -374,6 +372,7 @@ def extend_to_full_name(
     Returns:
         list[str]: list of the extended variable names
     """
+    dummy_variable = dummy_variable[0]
```

Comment:
if dummy_variable comes from list ([0]) is the type str (line 363 correct?)

Reply:
no it was not, should be a List[str]. Changed